### ISSUE TITLE
[Windows Terminal>Settings>Profile>Appearance] Name is not define for the 'Delete' button in 'Font Feature and Variable Font Axes' groups.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19689]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

**Repro Steps:**
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'Settings' menu item.
3.Settings page will open.
4.Now navigate to 'Profile' group and select 'default'.
5.Default page will open.
6.Now navigate to 'Appearance' section and expand it.
7.Now navigate to ''Font Feature and Variable Font Axes' groups and add some fonts.
4.Now turn on the narrator using 'CTRL+Win+Enter' keys.
4.Now navigate to the Delete' button.
5.Observe the issue here.

**User Experience:**
Screen reader users cannot identify the purpose of the button, making it difficult to delete font features or axes independently. This can lead to confusion and errors.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachements:**
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQAHoX2PwzioTI04iESK3JSrAXUREjoPQUOyihhwcWjEB8w?e=inWhHb

### Expected Behavior

Name should be define for the 'Delete button presnt in 'Font Feature and Variable Font Axes' groups.

### Actual Behavior

The Delete button does not have a programmatically determinable accessible name. Screen reader only announces it as “button”.

**Comment by carlos-zamora** [comment]
Just a matter of adding `AutomationProperties.Name` to the delete button
